src/plugins/rollbar.js

```diff
@@ -0,0 +1,48 @@
+import { version } from '_/package.json';
+
+import Rollbar from 'rollbar';
+
+// https://docs.rollbar.com/docs/rollbarjs-configuration-reference#notifier
+const config = {
+  accessToken: import.meta.env.VITE_ROLLBAR_ACCESS_TOKEN,
+  // Anonymize IP addresses
+  captureIp: 'anonymize',
+  // Capture uncaught exceptions and unhandled rejections
+  captureUncaught: true,
+  captureUnhandledRejections: true,
+  // Only allow items to be sent from certain domains
+  hostSafeList: [
+    'cuttle.cards',
+    ...(!import.meta.env.PROD) ? [
+      'localhost',
```

Comment:
I think we actually want to skip localhost unless we specifically decide to turn it on. That way we don't clutter the logs with development issues. 

Do you think the best way to handle that is simply to comment this out and update the config during build/boot if you want logging during localhost development? Or to use some kind of env var to enable/disable it?

Reply:
It won’t work unless you add the rollbar env, but if you need more security we can certainly just remove it or tie it to another to enable rollbar on local.

Reply:
I pushed a commit to just comment it out. This way we can opt-in to logging localhost errors but it's off by default